Cargo.toml

```diff
@@ -33,36 +33,36 @@ visit = ["swc_ecma_visit"]
 
 [dependencies]
 anyhow = { version = "1.0.64", optional = true }
-base64 = { version = "0.13.0", optional = true }
-data-url = { version = "0.1.1", optional = true }
-dprint-swc-ext = "0.4.0"
+base64 = { version = "0.13.1", optional = true }
+data-url = { version = "0.2.0", optional = true }
+dprint-swc-ext = "0.5.0"
 serde = { version = "1.0.144", features = ["derive"] }
 text_lines = { version = "0.6.0", features = ["serialization"] }
-url = { version = "2.2.2", features = ["serde"], optional = true }
+url = { version = "2.3.1", features = ["serde"], optional = true }
```

Comment:
@bartlomieju do you think upgrading url is ok?

Reply:
Yeah, I see no problem with it